src/game/server/gamemodes/insta_core/insta_core.cpp

```diff
@@ -31,6 +32,22 @@ CGameControllerInstaCore::CGameControllerInstaCore(class CGameContext *pGameServ
 	UpdateSpawnWeapons(true, true);
 	m_vFrozenQuitters.clear();
 	g_AntibobContext.m_pConsole = Console();
+
+	m_vMysteryRounds.clear();
+	if(Config()->m_SvMysteryRoundsFileName[0] != '\0')
+	{
+		CLineReader LineReader;
+		if(!LineReader.OpenFile(GameServer()->m_pStorage->OpenFile(Config()->m_SvMysteryRoundsFileName, IOFLAG_READ, IStorage::TYPE_ALL)))
+		{
+			dbg_msg("bombtag", "failed to open '%s'", Config()->m_SvMysteryRoundsFileName);
+			return;
+		}
+		while(const char *pLine = LineReader.Get())
+		{
+			if(str_length(pLine) && pLine[0] != '#')
```

Comment:
this looks like poor comment support. I suppose thats only an optimization  and not real comment support?
because this is a comment
`# foo`
but this is also a comment
`   # foo`